app/controllers/application_controller.rb

```diff
@@ -8,7 +8,6 @@ class ApplicationController < ActionController::Base
   before_action :set_locale
 
   private
-  
```

Comment:
Try adding back the extra space under `private` and deleting the `return` keyword (line will implicitly return)

Reply:
I did that still facing some issues @sashadev-sky